.github/ISSUE_TEMPLATE/data_issue_report.md

```diff
@@ -0,0 +1,39 @@
+---
+name: Dataset issue report
+about: Create a report about a problematic dataset
+title: 'Dataset problem: '
+labels: ''
+assignees: ''
+
+---
+
+**Describe the dataset issue**
+A clear and concise description of what the problem with the data is is. Here are some guidelines that you can use
+right out the box by filling in the blanks (note that, if needed we will assist you with writing a fix; also note that
+if the problem is with a CMIP6 dataset, we will alert the ESGF and/or the model developers for them to fix the problem
```

Comment:
It says here `we will alert the ESGF and/or the model developers`. I think it would be good to be more specific and say who will actually do this. Would you be willing to do this @valeriupredoi?

Reply:
good point! Yes, I can do that, but we should have at least one more person for the times when I'm at the pub :grin: How about @zklaus - he's respectable enough :grin:

Reply:
@zklaus You interested in doing this?

Reply:
Sure thing. :+1: Then we can also first evaluate complaints as to their merit. Will you put us two then as assignees in the issue template?

Reply:
@valeriupredoi Can you take care of updating the text so it mentions you two?

Reply:
done